### PR TITLE
[IOTDB-1969] Remove old metric framework from iotdb-server

### PR DESCRIPTION
**Pull request by stormbroken**
I remove old metric framework from iotdb-server, and all parameters about metric are in iotdb-metric.yml

### CODE REVIEW

File: site/src/main/.vuepress/config.js [hunk 1]
```diff
@@ -737,9 +737,8 @@ var config = {
 							['System-Tools/JMX-Tool','JMX Tool'],
 							['System-Tools/MLogParser-Tool','MLogParser Tool'],
 							['System-Tools/NodeTool','Node Tool'],
-							['System-Tools/Query-History-Visualization-Tool','Query History Visualization Tool'],
 							['System-Tools/Watermark-Tool','Watermark Tool'],
-							['System-Tools/Metric-Tool','Metric Tool']
+							['System-Tools/Metric-Tool','Metric-Tool']
```
Comment (SteveYurongSu) [inline site/src/main/.vuepress/config.js:741]:
```suggestion
							['System-Tools/Metric-Tool','Metric Tool']
```

Comment (stormbroken) [reply site/src/main/.vuepress/config.js]: Ok, I will fix this

File: server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java [hunk 2]
```diff
@@ -170,21 +93,12 @@
   private static final String INFO_INTERRUPT_ERROR =
       "Current Thread interrupted when dealing with request {}";
 
-  private static final int MAX_SIZE = CONFIG.getQueryCacheSizeInMetric();
-  private static final int DELETE_SIZE = 20;
-
-  private static final List<SqlArgument> sqlArgumentList = new ArrayList<>(MAX_SIZE);
-
   private long startTime = -1L;
```
Comment (SteveYurongSu) [inline server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java:96]:
```suggestion
```

Can the startTime be deleted?

Comment (stormbroken) [reply server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java:96]: Let me take a look

Comment (stormbroken) [reply server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java:96]: This parameters may used by tracing, so I can delete this.

Comment (SteveYurongSu) [reply server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java:96]: Ignore it :D

**Pull request merged**